bindings/Makefile

```diff
@@ -16,8 +17,13 @@ deps/$(CODEGEN_BIN):
 build/%: $(SWAGGER_SPEC) deps/$(CODEGEN_BIN)
 	mkdir -p build/$*
 	java -jar deps/$(CODEGEN_BIN) generate -i $(SWAGGER_SPEC) \
-		-l $* -o build/$*
+		-l $* -o build/$* -c config.json
 	touch build/$*
+	if [ $* = "python" ] ; then \
+		cp -rf ./build/python/determined/_swagger/client/ $(CLIENT_PATH); \
+		cp -rf ./build/python/determined._swagger.client/ $(CLIENT_PATH); \
+		test -z "$(shell git status --porcelain '$(CLIENT_PATH)')" ;\
```

Comment:
this belongs in a `check` target, e.g. `make build` should never fail just because there's new things to build, but the `make check` should fail if there are new things to build.

That `make check` target needs to get called from CI.  You'll have to look at the `lint-python` workflow for how that happens.